tools/config/__init__.py

```diff
@@ -780,7 +819,8 @@ def report(self):
         return {'app_config': self.app_config_location,
                 'library_configs': map(relpath, self.processed_configs.keys())}
 
-    def _generate_linker_overrides(self, rom_start, rom_size):
+    def _generate_linker_overrides(self, **active_rom_regions):
```

Comment:
Same suggestion as @theotherjimmy:
```suggestion
    def _generate_linker_overrides(self, rom_regions):
```